lib/src/postgrest_error.dart

```diff
@@ -25,4 +25,10 @@ class PostgrestError {
         'hint': hint,
         'code': code,
       };
+
+  @override
+  String toString() {
+    return 'PostgrestError{message: $message, '
+        'details: $details, hint: $hint, code: $code}';
```

Comment:
Could we make it slightly better by also applying `toString()` to details?

```suggestion
        'details: ${details?.toString()}, hint: $hint, code: $code}';
```

Reply:
this is unecessary tho. when $ is added, 'toString' to also called. also, the ?. operator isn't needed cuz we can also call toString() on null